unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEditorHUDV2ComponentView.cs

```diff
@@ -110,6 +116,21 @@ public void UpdateAvatarPreview(AvatarModel avatarModel)
             backpackPreviewPanel.SetLoadingActive(true);
         }
 
+        public void TakeSnapshotsAfterStopPreviewAnimation(IBackpackEditorHUDView.OnSnapshotsReady onSuccess, Action onFailed)
+        {
+            async UniTaskVoid TakeSnapshotsAfterStopPreviewAnimationAsync(CancellationToken ct)
+            {
+                ResetPreviewEmote();
+                await UniTask.Delay(MS_TO_RESET_PREVIEW_ANIMATION, cancellationToken: ct);
```

Comment:
I think it's risky to have the wait as an hardcoded value because if we change animation and it takes longer we risk having wrong screenshots, right?
But if there are no other ways it's okay

Reply:
This is the time needed to STOP the current animation and be sure that the avatar went back to the idle one, it will never take more than these 200 milliseconds.